Examples/Sources/MacroExamples/Implementation/Member/CaseDetectionMacro.swift

```diff
@@ -22,29 +22,28 @@ public enum CaseDetectionMacro: MemberMacro {
   ) throws -> [DeclSyntax] {
     declaration.memberBlock.members
       .compactMap { $0.decl.as(EnumCaseDeclSyntax.self) }
-      .map { $0.elements.first!.name }
-      .map { ($0, $0.initialUppercased) }
-      .map { original, uppercased in
+      .map { $0.elements }
+      .flatMap { $0 }
```

Comment:
Can this just be the following?

```suggestion
      .flatMap { $0.elements }
```